### DISCUSSION TITLE
Student Dev Pack

### DISCUSSION THREAD

**Discussion by Hudica** [community/community#110852] [GitHub Education]
### Hi everyone,

Hi, I am trying to upload proof of being a student but the only option i have is to take a photo from my computer camera and upload that, how can i actually upload a file to prove i am a student?

**Comment by thekarthikr** [comment]
Log into your GitHub account on your mobile device and take a picture of the proof using your mobile. That's how I did it.

**Comment by Yerock09** [comment]
Creo que si me pudieras dejar entrar pudiera

**Comment by notSaqlain** [comment]
Unfortunately, GitHub's current system for verifying student status in the Student Developer Pack application only allows capturing a picture with your webcam or using your device's camera. This means you cannot directly upload a file like a scanned document or transcript.

However, there are some workarounds you can try:

**1.**  **_Print your proof document and take a clear picture:_** Print out your proof of enrollment (e.g., school ID, enrollment letter) and capture a clear picture of the relevant information using your webcam or phone camera. Ensure good lighting and hold the document steady to make the text legible.

**2. _Use your phone's camera app and upload the photo:_** If your computer doesn't have a webcam, take a picture of your proof document using your phone's camera app. Then, transfer the photo to your computer and use the "Take a picture" option on the application page to upload it.

**3.** **_Contact GitHub support:_** If the above options are not feasible, you can reach out to GitHub support and explain your situation. They might be able to offer alternative solutions or provide guidance in specific cases. You can find their contact information on the GitHub help page: https://docs.github.com/.

While directly uploading a file isn't currently possible, these alternative methods should allow you to submit your proof of enrollment effectively.

**Please note:** If these solutions work and your application is successful, you can mark this discussion _as complete_ and close it.
